CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## [0.0.9+1]
+* `CapacityIndicator` colors can now be setted on its constructor
```

Comment:
"setted" isn't a word, change to something like "`CapacityIndicator` colors can now be set on its constructor"

Reply:
Oh sorry. I didn't know that

Reply:
No worries 😊